packages/apollo/src/commands/service/check.ts

```diff
@@ -283,22 +283,18 @@ export default class ServiceCheck extends ProjectCommand {
     const { isCi } = envCi();
 
     let schema: GraphQLSchema | undefined;
+    let graphID: string | undefined;
+    let graphVariant: string | undefined;
     try {
       await this.runTasks<TasksOutput>(
         ({ config, flags, project }) => {
-          if (!isServiceProject(project)) {
```

Comment:
this should still be here

Reply:
actually this needs to be removed otherwise when we use a user token as the engine_api_key this will throw